app/controllers/account_home_controller.rb

```diff
@@ -2,6 +2,6 @@ class AccountHomeController < ApplicationController
   include GovukPersonalisation::ControllerConcern
 
   def show
-    redirect_with_analytics GovukPersonalisation::Urls.your_account
+    redirect_with_analytics GovukPersonalisation::Urls.one_login_your_services
```

Comment:
Nice!

Reply:
(I'll mark all the positive comments as resolved).